Bindings/bindings.json

```diff
@@ -333,7 +342,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Service Bus connection",
-                    "help": "This is the connection string for your Service Bus account. Make sure that it has Send rights."
+                    "help": "This is the connection string for your Service Bus account. Make sure that it has Send rights.",
```

Comment:
Why are you guys lower casing these service names? It should be ServiceBus, just like we use DocumentDb, EventHub, etc.